### DISCUSSION TITLE
Converting age animations to global ones

### DISCUSSION THREAD

**Discussion by Hazado** [H-uru/Plasma#1182] [Ideas & Proposals]
So I know how to convert age animations to global animations.

This discussion is if we should and what animations are possible candidates.

Here is the animations I believe make good candidates for global animations.
```
Ahnonay     SwimDockExit
            ValveWheelCCW
            ValveWheelCW

Garrison    WallClimb*

Gira        VertBlast*

Teledahn    PowerTowerPrimerButton
            Hatch*
```

**Comment by dpogue** [comment]
I definitely support making the `WallClimb*` anims global (since the climbing brain tries to use them unconditionally and will blow up if they are not loaded).

I'm less convinced about some of the other Age-specific anims. They can be duplicated into other Ages if people want to use them.

**Comment by Hazado** [comment]
SwimDockExit is one that can be used to exit any body of water. Ive personally used it in relto and will be setting it up for ametist for use in her new age.

**Reply by dpogue** [reply]
yeah, that one probably makes sense to be global
